LoopKitUI/LoopUIPlugin.swift

```diff
@@ -6,22 +6,24 @@
 //  Copyright © 2019 LoopKit Authors. All rights reserved.
 //
 
-public protocol PumpManagerUIPlugin {
+import LoopKit
+
+public protocol PumpManagerUIPluginProvider {
```

Comment:
I think we should still call the modules that vend Pluggable classes "Plugins".  A "PluginProvider" sounds like something that vends different plugins.

Reply:
> So, I'm still trying to think of a way to provide all plugins.

I think that would be nice. What is preventing this?

Reply:
> > So, I'm still trying to think of a way to provide all plugins.
> 
> I think that would be nice. What is preventing this?

There is no interface to get all active plugins. Currently the plugins are spread across various classes.

Reply:
@ps2 Added something to the PR in Loop. Let me know what you think.